### PR TITLE
Various fixes for battery settings

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

File: pages/settings/devicelist/dc-in/PageAlternatorModel.qml [hunk 1]
```diff
@@ -85,61 +85,71 @@ ObjectModel {
...
 		onClicked: {
 			Global.pageManager.pushPage(overallHistoryComponent, { "title": text })
 		}
 
-		Component {
-			id: overallHistoryComponent
+		ObjectModelMonitor {
+			id: overallHistoryMonitor
+			model: overallHistoryModel
+		}
```
Comment (jpetrell) [inline pages/settings/devicelist/dc-in/PageAlternatorModel.qml:96]: Feels overly heavy solution, but I guess no way around that. Would maybe have opted for the empty state placeholder instead. The process of always initializing models for sub-pages seems architecture-wise not wise (code complexity, performance, responsibility encapsulation, etc.).

File: pages/settings/devicelist/battery/PageBatterySettings.qml [hunk 2]
```diff
@@ -112,7 +112,7 @@ Page {
 				}
 			}
 
-			ListRadioButtonGroupNoYes {
+			ListSwitch {
```
Comment (jpetrell) [inline pages/settings/devicelist/battery/PageBatterySettings.qml:91]: :partying_face: 

At some point could remove the 4 last remaining ListRadioButtonGroupNoYeses out there.

**Review by jpetrell — APPROVED** [review]

**Comment by jpetrell** [comment]
> At some point could remove the 4 last remaining ListRadioButtonGroupNoYeses out there.

Filed https://github.com/victronenergy/gui-v2/issues/865

File: pages/settings/devicelist/dc-in/PageAlternatorModel.qml (hunk 1 above)
Comment (blammit) [reply pages/settings/devicelist/dc-in/PageAlternatorModel.qml:96]: Agreed, I'd like to see what kind of alternative solutions we can come up with for this.

**Pull request merged**